http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1676,7 +1676,7 @@ paths:
           schema:
             type: string
       requestBody:
-        $ref: "#/components/requestBodies/CreateAppRequest"
+        $ref: "#/components/requestBodies/CreateAzureAppRequest"
```

Comment:
why are we further diverging V2 request models from being non Azure/google specific

Reply:
Is that an ideal toward which I should be working? Would renaming the new terms with "V2" instead of "Azure" be preferable?

Reply:
Went ahead and renamed

Reply:
Yeah, was a little unclear as to whether the model itself was azure-specific, so didn't want to be prescriptive. In general, I'd say we want to make v2 routes as non-azure-specific as possible as the eventual goal is to remove the google-project-dependent v1 routes and use workspaceId for all resources.